bylaws/5-decision-making.md

```diff
@@ -37,34 +33,34 @@ Proposals typically begin as GitHub Discussions for collaborative refinement.
 
 ### Sense Vote
 
+Members express their stance on proposals using *reactions* to express their level of comfort or agreement with a proposal.
```

Comment:
Express used twice in the same sentence 🤔

Reply:
<img width="38" alt="image" src="https://github.com/commonhaus/foundation/assets/808713/15089612-0278-46d5-986b-fb79994986fd">

Don't just tell me it's wrong, help fix it next time. Faster that way.

Reply:
I would if I knew how to fix it. I just know it looks weird. 

Me no speako English very good 😅